### PR TITLE
Slack Channel Meta overwrite documentation

### PR DESCRIPTION
**Pull request by jweingarten**
Add documentation for https://github.com/screwdriver-cd/notifications-slack/pull/28

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jweingarten** [comment]
@tkyi @jithine

File: docs/user-guide/metadata.md [hunk 1]
```diff
@@ -243,3 +243,28 @@ jobs:
 
 Result:
 ![notification-meta](./assets/notification-meta.png)
+
+#### Job-based Slack Channel
+*Note*: Job-based slack channel meta will only overwrite the basic slack notification channel. It is not a replacement for setting a [notification channel.](./configuration/settings.html#slack)
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
*Note*: Job-based Slack channel meta will only overwrite the basic Slack notification channel. It is not a replacement for setting a [notification channel](./configuration/settings#slack).
```

File: docs/user-guide/metadata.md [hunk 2]
```diff
@@ -243,3 +243,28 @@ jobs:
...
+
+Structure of meta variable is `notification.slack.<jobName>.channels`, replacing `<jobname>` with the name of the Screwdriver job.
+
+The setting is a comma separated string that allows setting multiple channels.
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
The setting is a comma-separated string that allows setting multiple channels.
```

File: docs/user-guide/metadata.md [hunk 3]
```diff
@@ -243,3 +243,28 @@ jobs:
...
+
+Example screwdriver.yaml notifying for job `component` different Slack channels :
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
Example screwdriver.yaml notifying different Slack channels upon job failure for the `component` job:
```

**Review by tkyi — APPROVED** [review]

**Pull request merged**